llpc/lower/llpcSpirvLowerRayQuery.cpp

```diff
@@ -864,7 +864,11 @@ template <> void SpirvLowerRayQuery::createRayQueryFunc<OpRayQueryGenerateInters
   storeAddr = m_builder->CreateGEP(
       rayQueryTy, rayQuery,
       {zero, m_builder->getInt32(RayQueryParams::Committed), m_builder->getInt32(RaySystemParams::RayTCurrent)});
-  m_builder->CreateStore(hitT, storeAddr);
+  Value *rayTMinAddr = m_builder->CreateGEP(rayQueryTy, rayQuery,
+                                            {m_builder->getInt32(0), m_builder->getInt32(RayQueryParams::RayTMin)});
```

Comment:
You can use CreateConstGEP2_32 here.

Reply:
Yeah, and everywhere else that accesses ray query member. I will do that in another PR.